tools/AzureRM/AzureRM.psd1

```diff
@@ -105,7 +105,7 @@ RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '4.1.1';
                @{ModuleName = 'AzureRM.Tags'; RequiredVersion = '4.0.0'; }, 
                @{ModuleName = 'AzureRM.TrafficManager'; RequiredVersion = '4.0.0'; }, 
                @{ModuleName = 'AzureRM.UsageAggregates'; RequiredVersion = '4.0.0'; }, 
-               @{ModuleName = 'AzureRM.Websites'; RequiredVersion = '4.0.0'; })
+               @{ModuleName = 'AzureRM.Websites'; RequiredVersion = '4.0.1-preview'; })
```

Comment:
Do not update AzureRM with this version - the breaking changes in this moduel cannot ship with AzureRM